charts/promtail/Chart.yaml

```
@@ -3,7 +3,7 @@ name: promtail
 description: Promtail is an agent which ships the contents of local logs to a Loki instance
 type: application
 appVersion: 2.7.2
-version: 6.8.2
+version: 6.8.3
```

New feature should bump to minor version.
```suggestion
version: 6.9.0
```

---

Should be good now @zanhsieh